prometheus_client/multiprocess.py

```diff
@@ -49,7 +49,22 @@ def merge(files, accumulate=True):
             typ = parts[0]
             multiprocess_mode = parts[1] if typ == Gauge._type else None
             pid = parts[2] if multiprocess_mode and len(parts) > 2 else None
-            d = MmapedDict(f, read_mode=True)
+            try:
+                d = MmapedDict(f, read_mode=True)
+            except EnvironmentError:
+                # The liveall and livesum gauge metrics, which only track
+                # metrics from live processes, are deleted when the worker
+                # process dies (mark_process_dead and, in postal-main,
+                # boot.gunicornconf.child_exit).
+                # Additionally, we have a single thread which will collect
+                # metrics files from dead workers, and merge them into a set of
+                # archive files at regular interviews (see
+                # multiprocess_exporter).
+                # Since collecting the files to
+                # merge and reading those files are non-atomic, it's very
+                # possible, and expected, that these files will not exist at
+                # this point
+                continue
```

Comment:
What other metrics come through here that we would want to raise on?

https://github.com/prometheus/client_python/pull/424/files#diff-52e98fdb8931e88f7d37feb48ba6ded2R64-R70